repl/src/main/scala/ammonite/repl/Main.scala

```diff
@@ -258,7 +265,10 @@ object Main{
       (fileToExecute, codeToExecute) match{
         case (None, None) => println("Loading..."); main.run()
         case (Some(path), None) =>
-          main.runScript(path, passThroughArgs, kwargs.toMap) match{
+          main.runScript(path,
+            passThroughArgs,
+            kwargs.toMap,
+            main.storageBackend) match{
```

Comment:
Closing paren on a new line, aligned with the first character of `main .runScript(`